soroban-sdk/src/crypto/bls12_381.rs

```diff
@@ -26,7 +77,7 @@ pub struct Bls12_381 {
 ///   `Fp`
 /// - The most significant three bits (bits 0-3) of the first byte are reserved
 ///   for encoding flags:
-///   - compression_flag (bit 0): Must always be set (1), as only uncompressed
+///   - compression_flag (bit 0): Must always be unset (0), as only uncompressed
```

Comment:
Is this behaviour that's changing that's related to this change, or a bug in documentation? If latter, and since this looks isolated from everything else here, could you make this fix in a separate PR? so that it is captured clearly in the changelog as its own line item, and in general to increase the visibility.

Reply:
it is a bug in the documentation. separated out the fix in https://github.com/stellar/rs-soroban-sdk/pull/1455